cmd/influxd/run/server_test.go

```diff
@@ -1010,6 +1010,9 @@ func NewConfig() *run.Config {
 	c.HTTPD.Enabled = true
 	c.HTTPD.BindAddress = "127.0.0.1:0"
 	c.HTTPD.LogEnabled = testing.Verbose()
+
+	c.UDP.BindAddress = "127.0.0.1:0"
+	c.UDP.Database = "db0"
```

Comment:
We actually don't want to add these.  This would enable the UDP endpoint for every test.  We use this function to create a `base` config for the tests to use, and then turn on specific endpoints in the tests themselves.  Simply revert the change to this file and we should be fine.